client/src/components/SelectionDialog/SelectionDialog.vue

```diff
@@ -1,32 +1,44 @@
 <template>
-    <b-modal
-        modal-class="selection-dialog-modal"
-        v-if="modalShow"
-        visible
-        ok-only
-        ok-title="Close"
-        :static="modalStatic"
-        @hide="hideModal">
+    <b-modal modal-class="selection-dialog-modal" v-if="modalShow" visible :static="modalStatic" @hide="hideModal">
         <template v-slot:modal-header>
             <slot name="search"> </slot>
         </template>
+        <b-alert v-if="showFtpHelper" variant="info" show>
+            This Galaxy server allows you to upload files via FTP. To upload some files, log in to the FTP server at
+            <strong>{{ ftpUploadSite }}</strong> using your Galaxy credentials. For help visit the
+            <a href="https://galaxyproject.org/ftp-upload/" target="_blank">tutorial</a>.
+            <span v-if="oidcEnabled"
+                ><br />If you are signed-in to Galaxy using a third-party identity and you
+                <strong>do not have a Galaxy password</strong> please use the reset password option in the login form
+                with your email to create a password for your account.</span
+            >
+        </b-alert>
```

Comment:
I agree with you. I created a new slot named `helper` for wrapping any help text for any resource. It would be nice to have a helper option in the `config/file_sources_conf.yml` config file.

Reply:
Thank you!, I agree, the `showFtpHelper` could be even more generic to show other info messages from other file sources.